docs/celo-codebase/protocol/bridging/bridging-native-assets.md

```diff
@@ -1,60 +1,63 @@
 ---
 title: Bridge Native Assets with Etherscan
+description: How to bridge native assets from ETH and Matic to Celo with Etherscan.
 ---
 
-Bridging ETH and Matic to Celo.
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
 
----
-
-## How to complete this guide
-
-* **Step 1:** Call the EthHelper (to send tokens)
-* **Step 2:** Wait
+# Bridge Native Assets with Etherscan
 
----
-
-## Step 1: Call the EthHelper
+How to bridge native assets from ETH and Matic to Celo with Etherscan.
 
-1. Navigate to the [Etherscan](https://etherscan.io/) page for the EthHelper.
+___
+## Call the EthHelper
 
-:::info
+Navigate to the [Etherscan](https://etherscan.io/) page for the EthHelper.
```

Comment:
What do you think about updating the link to direct to the appropriate smart contract for each network and moving the link into the tab for that network?

Reply:
i made the update